examples/site.yml

```diff
@@ -3,13 +3,12 @@
 
   pre_tasks:
     - name: Update APT cache
-      apt: update_cache=yes
+      apt:
+        update_cache: yes
 
   roles:
-    - { role: "azavea.python-security", when: 'ansible_python_version | version_compare("2.7.9", "<")' }
     - { role: "azavea.docker" }
 
   tasks:
     - name: Bring up a test container
       command: docker run busybox sh
-      become: yes
```

Comment:
`become` is being set at the Vagrant project level, and is true for the entire playbook.